modules/helm-installing-a-helm-chart-on-an-openshift-cluster.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id="installing-a-helm-chart-on-an-openshift-cluster_{context}"]
+
+= Installing a Helm chart on an {product-title} cluster
+
+.Prerequisites
+* You have a running {product-title} cluster and you have logged into it.
+* You have installed Helm.
+
+.Procedure
+
+. Create a new project:
++
+----
+$ oc new-project mysql
+----
+
+. Add a repository of Helm charts to your local Helm client:
++
+----
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+"stable" has been added to your repositories
+----
+
+. Install an example MySQL chart:
++
+----
+$ helm install example-mysql stable/mysql
```

Comment:
Is it verified that the chart installs successfully on OpenShift? The pods fails to come up when I try it

Reply:
@siamaksade Yes It works on OpenShift cluster, I tried it and pod did show up for me. check https://console-openshift-console.apps.olemefer-164.devcluster.openshift.com/k8s/cluster/projects/akash-helm-server/workloads (Link will be invalid after cluster goes away).